pinot-tools/src/main/java/org/apache/pinot/tools/MultistageEngineQuickStart.java

```diff
@@ -94,6 +94,11 @@ public Map<String, Object> getConfigOverrides() {
     return overrides;
   }
 
+  @Override
+  protected int getNumQuickstartRunnerServers() {
+    return 4;
```

Comment:
FYI. this doesn't change the e2e test servers. just in case you were changing this for the e2e test